### ISSUE TITLE
button adds default onclick even when user defined onclick is added

### ISSUE THREAD

**Issue by LupusUmbrae** [TheCoder4eu/BootsFaces-OSP#211]
Not sure it's technically a bug but when i set an onclick on a b:button it auto suffixes it with a "window.location.href=", I've had a look at the code and cannot see anyway to prevent it from appending this JS.

JSF:
```
<b:button value="Update Config" onclick="updateConfig();"/>
```

HTML:
```
<button id="graphconfig:j_idt58" name="graphconfig:j_idt58" type="button" class="btn btn-default" onclick="updateConfig();window.location.href='';">Update Config</button>
```

**Comment by LupusUmbrae** [comment]
Should note I'm on version 0.7.0, just had a look at the 0.8.0 tag and looks like this has been fixed in that version..

**Comment by LupusUmbrae** [comment]
Can confirm this is fixed in 0.8.0 thanks
